go/enclave/db/storage.go

```diff
@@ -588,7 +658,7 @@ func (s *storageImpl) FilterLogs(requestingAccount *gethcommon.Address, fromBloc
 	return s.loadLogs(requestingAccount, query, queryParams)
 }
 
-func hash(ns sql2.NullString) gethcommon.Hash {
+func hashString(ns sql2.NullString) gethcommon.Hash {
```

Comment:
strToHash and bytesToHash may be clearer names?

Reply:
Updated both methods to be clearer.